js/analyzer.js

```diff
@@ -67,9 +67,12 @@ Analyzer.prototype.checkConfig = function() {
  * YoastSEO.Analyzer initialization. Loads defaults and overloads custom settings.
  */
 Analyzer.prototype.init = function( args ) {
+	var metaValues = {
```

Comment:
I vote for `args.paper = new Paper( args.keyword, { metaDescription: args.meta } )`. 🚲🏠